### DISCUSSION TITLE
Newbie needs help

### DISCUSSION THREAD

**Discussion by jamjamguy** [cwendt94/espn-api#210] [General]
I am the definition of a newb. Do I need to fork this to use it for my own league?

Where do I paste the code in to start getting this to work? :/ 

Hopefully that can be answered simply! lol

**Comment by cwendt94** [comment]
So first you will want to make sure you have python and pip installed on your computer, [here](https://packaging.python.org/guides/installing-using-pip-and-virtual-environments/) is the official documentation.

After you have that setup you can install the package using pip: `pip install espn_api`

Now you can use the package in the python cli or file. The [Wiki](https://github.com/cwendt94/espn-api/wiki) goes through getting started and the different functions you can call.
